content/Guides/DistributedComplianceLedger.md

```diff
@@ -0,0 +1,142 @@
++++
+title = "Distributed Compliance Ledger (DCL)"
+
+chapter = false
+weight = 10
++++
+
+
+## Table of Contents
+- [DCL Introduction](#dcl-introduction)
+  - [Types of nodes that operate in the DCL](#types-of-nodes-that-operate-in-the-dcl)
+  - [Types of account roles that can interact with the DCL](#types-of-account-roles-that-can-interact-with-the-dcl)
+  - [Main-Net and Test-Net?](#main-net-and-test-net)
+  - [Access and Interaction with the DCL](#access-and-interaction-with-the-dcl)
+  - [Getting an account for writing to the DCL](#getting-an-account-for-writing-to-the-dcl)
+  - [Interacting with the DCL](#interacting-with-the-dcl)
+    - [Web User Interface](#web-user-interface)
+    - [Command Line Interface](#command-line-interface)
+  - [Observer Nodes](#observer-nodes)
+    - [Main-Net public ONs](#main-net-public-ons)
+    - [Test-Net public ONs](#test-net-public-ons)
+- [FAQs](#faqs)
+  - [Why is my Vendor information not listed in the DCL?](#why-is-my-vendor-information-not-listed-in-the-dcl)
+  - [When should I write my product information to the Main-Net DCL?](#when-should-i-write-my-product-information-to-the-main-net-dcl)
+- [Contact](#contact)
+
+## DCL Introduction
+The DCL is a blockchain-based system owned and hosted by Alliance members. It is used by the Matter protocol for storing information such as:
+
+| Information          | Description                                               | Account Role involved                                                                    |
+| -------------------- | --------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
+| Vendor               | Information about the manufacturer or vendor of a device. | Data is entered by an approved Vendor account.                                             |
+| Product              | Information about the product device including description, software version, OTA information, etc. | Data is entered by an approved Vendor account.   |
+| Compliance status    | Information about the certification status including certification ID, certification date, etc. | Data is entered by the Alliance's Certification Center account.|
+| PAA Root CAs         | Information about Product Attestation Authorities (PAA) Root CAs used by commissioners during Device Attestation process. | Data is proposed by the Alliance's Trustee and approved by other DCL Trustees.|

```

Comment:
For the Description field we could drop "Information about" on all of them.  So just say

"The manufacturer or vendor of a device."
"The product device..."

Etc

Reply:
Done